components/prombench/manifests/results/alertmanager-github-receiver.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertmanager-github-receiver
+spec:
+  replicas: 1 
+  selector:
+    matchLabels:
+      app: alertmanager-github-receiver
+  template:
+    metadata:
+      labels:
+        app: alertmanager-github-receiver
+    spec: 
+      containers:
+      - name: alertmanager-github-receiver
+        image: docker.io/krasimir/alertmanager-github-receiver
```

Comment:
where can i find more about this docker image? is this the modified version of https://github.com/m-lab/alertmanager-github-receiver ? I am unable to understand how it figures out the issue to comment on.

link to docker hub: https://hub.docker.com/r/krasimir/alertmanager-github-receiver

cc @krasi-georgiev @sipian

Reply:
I had a quick look and can't figure it out either.
It should be just few lines to add this functionality and than can open a pr against `m-lab/alertmanager-github-receiver` to add it upstream.